test/mock-agent.js

```diff
@@ -2494,3 +2494,30 @@ test('MockAgent - headers in mock dispatcher intercept should be case-insensitiv
 
   t.end()
 })
+
+test('MockAgent - headers should be array of strings', { skip: nodeMajor < 16 }, async (t) => {
```

Comment:
why skip is Node is < 16?

Reply:
just an oversight, not necessary to skip it